imap/conversations.h

```diff
@@ -289,6 +289,12 @@ extern int conversations_guid_cid_lookup(struct conversations_state *state,
    strcmpsafe(conv_guidrec_uniqueid(rec), mailbox_uniqueid(mbox)) : \
    strcmpsafe(conv_guidrec_mboxname(rec), mailbox_name(mbox)) \
  )
+
+/* J record items */
+#define CONV_JMAPID_SIZE 11  // 64-bits base64-encoded w/o padding
+extern int conversations_lookup_jmapid(struct conversations_state *state,
```

Comment:
The other lookup function in conversations.h is named `conversations_guid_cid_lookup`, so should this be named consistently, e.g. as `conversations_jmapid_guidrep_lookup`? Not that I think that would be a great name, but I would prefer the API to use consistent naming either way.

Reply:
Sure.  I can rename it.